front/src/applications/operationalStudies/components/Scenario/ScenarioContent.tsx

```diff
@@ -259,19 +259,29 @@ const ScenarioContent = ({
             </BoardWrapper>
           </div>
         </div>
-        <div
-          style={{ display: activeBoards.has('conflicts') ? 'block' : 'none' }}
-          className="right-column"
-          data-testid="conflicts-list"
+        <BoardWrapper
+          hidden={!activeBoards.has('conflicts')}
+          name={`${conflicts?.length ?? ''} CONFLICTS`}
         >
-          {conflicts && (
-            <ConflictsList
-              conflicts={conflicts}
-              timetableItems={timetableItemsWithDetails}
-              onConflictClick={handleConflictClick}
-            />
-          )}
-        </div>
+          <div className="right-column" data-testid="conflicts-list">
```

Comment:
Shouldn't we put the div with `className="right-column"` as a parent of the BoardWrapper and not the other way around?"

Reply:
if we put `className="right-column"` as a parent, the show hide won't remove the column

<img width="1199" height="492" alt="Capture d'écran 2025-07-28 142212" src="https://github.com/user-attachments/assets/46451b0b-d934-4e9a-824c-e307577e1344" />

Reply:
We could do like what we did for left-column:

```suggestion
          <div
          className="right-column"
          data-testid="conflicts-list"
          style={{ display: activeBoards.has('conflicts') ? 'block' : 'none' }}
        >
          <BoardWrapper name={`${conflicts?.length ?? ''} CONFLICTS`}>
            {!conflicts && (
              <Loader
                msg={t('main.loadingConflicts')}
                className="scenario-loader"
                childClass="scenario-loader-msg"
              />
            )}
            {conflicts && (
              <ConflictsList
                conflicts={conflicts}
                timetableItems={timetableItemsWithDetails}
                onConflictClick={handleConflictClick}
              />
            )}
          </BoardWrapper>
        </div>
```